src/core/net/ip6.hpp

```diff
@@ -419,6 +439,10 @@ class Ip6 : public InstanceLocator, private NonCopyable
     Tcp mTcp;
 #endif
 
+#if OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE
+    static bool mTmfOriginFilterEnabled;
+#endif
```

Comment:
Can move the definition after `bool mIsReceiveIp6FilterEnabled`? (for better struct packing?)